app/robots.ts

```diff
@@ -0,0 +1,12 @@
+import { MetadataRoute } from "next";
+
+export default function robots(): MetadataRoute.Robots {
+  return {
+    rules: {
+      userAgent: '*',
+      allow: '/',
+      disallow: ['/api/', '/admin/'],
+    },
+    sitemap: 'https://noblocks.xyz/sitemap.xml',
```

Comment:
/sitemap.xml is not a valid route... should we create a sitemap or how does sitemap.ts serve here?

Reply:
the `sitemap.ts` is used to auto-generate the `sitemap.xml`. [docs](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/sitemap#generating-a-sitemap-using-code-js-ts)